bats/tests/helpers/vm.bash

```diff
@@ -378,6 +382,23 @@ wait_for_container_engine() {
     try --max 12 --delay 10 get_container_engine_info
 }
 
+# Wait fot the extension manager to be initialized.
+wait_for_extension_manager() {
+    trace "waiting for extension manager to be ready"
+    # We want to match specific error strings, so we can't use try() directly.
```

Comment:
The usual pattern would be to create an `assert_extension_manager_ready` function, and then call that function using `try`.

Reply:
Nm, I see what you mean: you want to retry only on specific failures, but not others.